### DISCUSSION TITLE
Jacktrip p2p connection enables mic input to be connected.

### DISCUSSION THREAD

**Discussion by neosdeah** [jacktrip/jacktrip#1280] [General]
Hi, 

I'm using jacktrip to stream/send audio from a windows machine to my Fedora machine. That works good. But one odd thing happens: When the connection is established, I can see in qjackctl that my microphone (capture) is automatically being connected to the jacktrip - send 1 channel. When I disconnect that particular link, the audio gets all garbled.

Is that by design? Or is there a way I can prevent the microphone from being connected to the jacktrip audio?
![Jacktrip](https://github.com/jacktrip/jacktrip/assets/112099829/786d262a-e9ec-4b80-89c5-3e5b97d01bde)

**Comment by cchafe** [comment]
See if the cli argument "--nojackportsconnect" does what you intend.
